docs/content/kv_commands.md

```diff
@@ -340,8 +340,12 @@ Walks the given directory and runs a bulk upload, using the path to an asset as
 
 #### Usage
 
-```sh
-$ wrangler kv:bucket upload f7b02e7fc70443149ac906dd81ec1791 ./public
+```console
+$ wrangler kv:bucket upload --binding MY_KV ./public
```

Comment:
this PR made me notice: some of our examples use `MY_KV` and others use `KV`. Let's make them consistent. I think `MY_KV` works nicely, as it makes more clear that this is a custom name, not a standard name.